### DISCUSSION TITLE
How to change the cascader options style?

### DISCUSSION THREAD

**Discussion by chioio** [tusen-ai/naive-ui#3665] [Q&A] [answered]
I want to hide the top level options checkbox to prevent check the all children, but keep multiple select in children options.
<img width="606" alt="image" src="https://user-images.githubusercontent.com/36913769/188145693-3e0e9f8a-8bd7-4ba6-80f4-c843c1f203e0.png">

**Comment by Sepush** [comment, marked as answer]
use ThemeOverrides you can find the useage on the doc
https://www.naiveui.com/en-US/os-theme/docs/customize-theme
https://www.naiveui.com/en-US/os-theme/docs/theme
BTW you could use theme editor to preview like the following
![image](https://user-images.githubusercontent.com/39197136/189376154-18738d37-72d3-4e27-a1f6-08ea188b97be.png)
